modules/broker-cli-client/src/main/java/io/ballerina/messaging/broker/client/utils/Constants.java

```diff
@@ -30,6 +30,7 @@
     public static final String DEFAULT_PASSWORD = "admin";
     public static final String BROKER_CONNECTION_URL_SUFFIX = "/broker/v1.0/";
     public static final String DEFAULT_CONFIG_FILE_PATH = "cli-config.yml";
+    public static final String ROOT_COMMAND = "broker-admin";
```

Comment:
Instead of hard-coding this value can we make the script pass the value? Maybe we have to set an environment variable for that.

Reply:
+1